aspnetcore/blazor/handle-errors.md

```diff
@@ -168,9 +168,9 @@ When a circuit ends because a user has disconnected and the framework is cleanin
 
 Blazor components can be prerendered using `Html.RenderComponentAsync` so that their rendered HTML markup is returned as part of the user's initial HTTP request. This works by:
 
-* Creating a new circuit containing all of the prerendered components that are part of the same page.
+* Creating a new circuit to update the prerendered components that are part of the same page.
```

Comment:
Hmm, we don't really update the components. We render them into the page as part of the response. Honestly, the relationship between the circuit and the prerendered components isn't super relevant.

Reply:
How about this?

```suggestion
* Creating a new circuit for all of the prerendered components that are part of the same page.
```